config/cmake/FindBoost.cmake

```diff
@@ -0,0 +1,2102 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
Why is find boost included? This is part of cmake?  I don't want to be responsible for maintaining any updates when Boost or cmake is updated.